editor/components/post-author/check.js

```diff
@@ -11,17 +11,23 @@ import { compose } from '@wordpress/element';
 
 export function PostAuthorCheck( { user, users, children } ) {
 	const authors = filter( users.data, ( { capabilities } ) => capabilities.level_1 );
-	if ( ! user.data || ! user.data.capabilities.publish_posts || authors.length < 2 ) {
+	const userCaps = user.data ?
+		{ ...user.data.capabilities, ...user.data.post_type_capabilities } :
+		{ 'publish_posts': false };
```

Comment:
Do we need if this will be always a "post type capability" or not? In which case we coud avoid using `user.data.capabilities` entirely?

Minor: maybe we could avoid creating a separate object, by leveraging lodash `get`

`const canPublishPosts = get( user, 'data.post_type_capabilities.publish_posts',  false )`

Reply:
Good point, i think we don't need it, i'll update the PR in this direction + thanks for the tip about lodash :+1: